Library/Homebrew/formula_creator.rb

```diff
@@ -201,8 +201,14 @@ def install
             virtualenv_install_with_resources
         <% elsif @mode == :ruby %>
             ENV["GEM_HOME"] = libexec
+
+            # resources.each do |r|
+            #   system "gem", "install", r.cached_download, "--ignore-dependencies",
+            #      "--no-document", "--install-dir", libexec
+            # end
+
```

Comment:
This should almost never be needed for Ruby and `bundle install` should be used instead.

Reply:
@BingoKingo Yes, that should be removed, thanks.